ui/src/services/NodeVolumesUtilsData.js

```diff
@@ -0,0 +1,2371 @@
+export const state_volume_filtered_by_node_master0 = {
```

Comment:
How is this variable describing "volumes filtered by node master0"? It looks to me as a rather "normal" application state, with three nodes and volumes on each node...